ext/reflection/php_reflection.c

```diff
@@ -2996,7 +2996,7 @@ ZEND_METHOD(reflection_type, isBuiltin)
 	}
 	GET_REFLECTION_OBJECT_PTR(param);
 
-	RETVAL_BOOL(param->arg_info->type_hint != IS_OBJECT);
```

Comment:
It's not necessary to cast to `zend_internal_arg_info` here, you can just write `param->arg_info->class_name == NULL`.